README.md

```diff
@@ -8,29 +9,119 @@
 
 RIGImageGallery is an image gallery for iOS written in Swift.
 
-The goal is to offer sensible defaults that takes care of most of building an image gallery automatically, and offer easy, block-based customization.
+This library is part of the Raizlabs Interface Guidelines, which are  UI components that offer sensible defaults to help a project get off the ground quickly with components that feel native to the platform, and with easy to use customization options.
+
+RIGImageGallery is a photo gallery meant to provide most of the functionality of the image gallery in the system Photos app, and handle asynchronous loading of images.
+
+## Features
+
+- [x] Swipe to advance
+- [x] Pinch to zoom
+- [x] Double tap to toggle 1:1 zoom
+- [x] Single tap to hide the nav bar
+
+## Requirements
+
+- iOS 9.0+
+- Xcode 8.0+
 
 ## Installation with CocoaPods
 
-RIGImageGallery is available through [CocoaPods](http://cocoapods.org). To install
-it, simply add the following line to your Podfile:
+#### CocoaPods
+You can use [CocoaPods](http://cocoapods.org) to install `RIGImageGallery` by adding it to your `Podfile`:
 
 ```ruby
+platform :ios, '9.0'
+use_frameworks!
 pod 'RIGImageGallery'
```

Comment:
#pp I think I made this comment elsewhere, maybe on a different project, but doesn't a valid Podfile need a target specifier? Rather than getting wordy, I'd keep it simple and just have the single `pod 'RIGImageGallery'` line.